string address
float32 xbee_trans_speed
